rollup.config.js

```diff
@@ -27,15 +26,36 @@ const isCJS = format === 'cjs';
 
 const babelOptions = getBabelPreset();
 
+/**
+ * Note:
+ *   In order to avoid bundling dependencies here and within consumers
+ *   we consider them external as they will be "bundled" by the consumers bundler (e.g. webpack) or
+ *   resolved by Node.js anyway.
+ *
+ *   However, some dependencies are made available through flopflip has they can not
+ *   be bundled twice in different versions. Something which can easily happen and causes hard
+ *   to discover bugs. THose dependencies are not considered to be external.
+ */
+const dependenciesRequiringToBeBundled = ['@flopflip/react-broadcast'];
+const pkgDependencies = Object.keys(pkg.dependencies || {});
+const pkgPeerDependencies = Object.keys(pkg.peerDependencies || {});
+const pkgOptionalDependencies = Object.keys(pkg.optionalDependencies || {});
+
+const externalDependencies = pkgDependencies
+  .concat(pkgPeerDependencies)
+  .concat(pkgOptionalDependencies)
+  .filter(
+    potentiallyExternalDependency =>
+      !dependenciesRequiringToBeBundled.includes(potentiallyExternalDependency)
+  );
+
```

Comment:
I hope the naming makes sense. I don't mind having it here straight in the config to make it explicit.